apps/api/src/routes/applicationRouter.ts

```diff
@@ -1024,7 +1024,7 @@ applicationRouter.post(
 
 applicationRouter.get(
 	'/:applicationId/revisions',
-	authMiddleware({ requiredRoles: ['APPLICANT', 'DAC_MEMBER'] }),
+	authMiddleware({ requiredRoles: ['APPLICANT', 'INSTITUTIONAL_REP', 'DAC_MEMBER'] }),
```

Comment:
Pretty sure that you can skip `requiredRoles` when it is all roles (other than anonymous).